### ISSUE TITLE
FullyQualifiedClassNameInAnnotation false positive

### ISSUE THREAD

**Issue by dereuromark** [slevomat/coding-standard#1455]
```
FOUND 2 ERRORS AFFECTING 1 LINE
--------------------------------------------------------------------------------
 46 | ERROR | [x] Class name \Cake\ORM\ in @method should be referenced via a
    |       |     fully qualified name.
    |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
 46 | ERROR | [x] Class name \Cake\ORM\ in @method should be referenced via a
    |       |     fully qualified name.
    |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
-----------------------------------------
```
On
```php
 * @method \Cake\Collection\CollectionInterface sortBy(callable|string $path, int $order = \SORT_DESC, int $sort = \SORT_NUMERIC) Sorts the query with the callback
 ```
 
 With or without leading \ CS is not happy
 
 Refs https://github.com/cakephp/cakephp/pull/16822 and CI for proof

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/41196b4fa281035cad5b484761689f061870c04f
